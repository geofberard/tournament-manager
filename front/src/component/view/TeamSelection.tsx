import { Card, CardContent, CardMedia, CircularProgress, Grid } from "@mui/material";
import * as React from "react";
import { FC, Suspense } from "react";
import { TeamSelector } from "../atom/TeamSelector";

export const TeamSelection: FC = () => {
    return (
        <Grid container item md={7}
            flexGrow={1}
            direction="column"
            justifyContent="center"
            alignItems="center">
            <Card sx={{ maxWidth: "400px" }} elevation={5}>
                <CardMedia
                    component="img"
                    alt="VolleyBall"
                    image="img/scuf-miniature.png"
                    title=""
                />
                <CardContent sx={{ textAlign: "center" }}>
                    <Suspense fallback={<CircularProgress />} >
                        <TeamSelector />
                    </Suspense>
                </CardContent>
            </Card>
        </Grid>
    );
};
