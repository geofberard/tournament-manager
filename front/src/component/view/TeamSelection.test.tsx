import * as React from 'react';
import { testSnapshot } from '../../test/TestUtils';
import { TeamSelection } from './TeamSelection';

testSnapshot(<TeamSelection />)