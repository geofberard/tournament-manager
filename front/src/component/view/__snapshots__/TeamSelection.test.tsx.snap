// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match last snapshot 1`] = `
<ForwardRef(Grid)
  alignItems="center"
  container={true}
  direction="column"
  flexGrow={1}
  item={true}
  justifyContent="center"
  md={7}
>
  <ForwardRef(Card)
    elevation={5}
    sx={
      Object {
        "maxWidth": "400px",
      }
    }
  >
    <ForwardRef(CardMedia)
      alt="VolleyBall"
      component="img"
      image="img/scuf-miniature.png"
      title=""
    />
    <ForwardRef(CardContent)
      sx={
        Object {
          "textAlign": "center",
        }
      }
    >
      <React.Suspense
        fallback={<ForwardRef(CircularProgress) />}
      >
        <TeamSelector />
      </React.Suspense>
    </ForwardRef(CardContent)>
  </ForwardRef(Card)>
</ForwardRef(Grid)>
`;
